recipes/elk/webview.js

```diff
@@ -0,0 +1,17 @@
+module.exports = (Ferdium) => {
+    // Inject css
+    // Ferdium.injectCSS(_path.default.join(__dirname, 'service.css'));
+
+    // Get notifications
+    Ferdium.loop(() => {
+        const notifications = document.querySelectorAll("[href$=notifications]");
+        if (notifications.length === 0) {
+          return;
+        }
+
+        // Assume first element contains the number of notifications
+        let parsedValue = Ferdium.safeParseInt(notifications[0].outerText);
```

Comment:
Is it possible for `notifications[0]` to be `undefined` or `null`? If so, you will still need to do some defensive programming here imo